test/suite/test_backup15.py

```diff
@@ -278,38 +281,12 @@ def add_data_validate_backups(self):
 
         # Each call now to take a full backup will make a copy into a full directory. Then
         # each incremental will take an incremental backup and we can compare them.
-        self.add_data(self.uri)
-        self.session.checkpoint()
-        self.take_full_backup()
-        self.take_incr_backup()
-        self.compare_backups(self.uri)
-
-        self.add_data(self.uri)
-        self.session.checkpoint()
-        self.take_incr_backup()
-        self.take_full_backup()
-        self.compare_backups(self.uri)
-
-        self.add_data(self.uri)
-        self.session.checkpoint()
-        self.take_full_backup()
-        self.take_incr_backup()
-        self.compare_backups(self.uri)
-
-        self.add_data(self.uri)
-        self.take_incr_backup()
-        self.take_full_backup()
-        self.compare_backups(self.uri)
-
-    def test_backup15(self):
-        os.mkdir(self.bkp_home)
-        self.home = self.bkp_home
-        self.session.create(self.uri, "key_format=S,value_format=S")
-
-        self.setup_directories()
-
-        self.pr('*** Add data, checkpoint, take backups and validate ***')
-        self.add_data_validate_backups()
```

Comment:
The original code does `take_full_backup` and `take_incr_backup` in a different order every other time.  Don't know if that makes for a better test?